src/app/grid/grid-dynamic-chart-data/directives/chart-integration/initializers.ts

```diff
@@ -74,28 +82,31 @@ export interface IStackedFragmentOptions {
     [key: string]: any;
 }
 
-export interface IChartComponentOptions {
-    chartOptions?: IChartOptions;
-    seriesOptions?: IChartSeriesOptions[];
-    xAxisOptions?: IXAxesOptions;
-    yAxisOptions?: IYAxesOptions;
-    stackedFragmentOptions?: IStackedFragmentOptions;
+export class ChartComponentOptions {
```

Comment:
Are you sure that we need to switch this interface to class, according to the way you use it in you code?!